modules/ept-frontend/client/components/post_processing/post-processing.directive.js

```diff
@@ -89,11 +89,7 @@ module.exports = ['$timeout', '$filter', '$compile', function($timeout, $filter,
         var classString = styleString.replace('class="bbcode-', '');
         classString = classString.replace('"', '');
 
-        if (classString.indexOf('color-_') === 0) {
```

Comment:
Just curious, I didn't write this piece of code, but was this not doing anything?